pkg/operation/botanist/addons.go

```diff
@@ -43,7 +43,22 @@ func (b *Botanist) DestroyIngressDNSRecord() error {
 // GenerateNginxIngressConfig generates the values which are required to render the chart of
 // the nginx-ingress-controller properly.
 func (b *Botanist) GenerateNginxIngressConfig() (map[string]interface{}, error) {
```

Comment:
Any change here is not needed (actually, can you please remove the whole method (it is not called anywhere and a left over of previous versions)).